README.md

```diff
@@ -1,2 +1,98 @@
 # Winforms-map-getting-started
-This repository contains samples for winforms map getting started
+The [WinForms Map control](https://www.syncfusion.com/winforms-ui-controls/map?utm_source=github&utm_medium=listing&utm_campaign=winforms-diagram-github-samples) is a powerful data visualization component that displays statistical information for a geographical area. It has highly interactive and customizable features such as zooming, panning, selecting, markers, bubbles, and color mapping. Using the Map control, you can render various types of maps, such as sales, political-boundary, weather, electoral, and route.
```

Comment:
change the UTM param in all places
"?utm_source=github&utm_medium=listing&utm_campaign=winforms-map-github-samples"

Reply:
Resolved

Reply:
resolved